planet/src/main/java/com/google/planet/data/ItineraryGenerator.java

```diff
@@ -17,10 +17,64 @@
 import java.util.List;
 import java.util.Arrays;
 import java.util.ArrayList;
+import java.util.Collections;
 
 public final class ItineraryGenerator {
     public List<ItineraryItem> generateItinerary(List<Event> events) { 
```

Comment:
Is it necessary for all these methods to be public? It seems that you are only testing generateItinerary(), it'd make sense to mark everything else private.

Reply:
Okay!